receiver/sqlqueryreceiver/metrics.go

```diff
@@ -29,6 +29,28 @@ func rowToMetric(row stringMap, cfg MetricCfg, dest pmetric.Metric, startTime pc
 	dest.SetUnit(cfg.Unit)
 	dataPointSlice := setMetricFields(cfg, dest)
 	dataPoint := dataPointSlice.AppendEmpty()
+	if cfg.StartTsColumn != "" {
+		if val, found := row[cfg.StartTsColumn]; found {
+			timestamp, err := strconv.ParseInt(val, 10, 64)
```

Comment:
Can we replace these numbers with constants (here and on line 45)?

Reply:
The following has been used to keep it consistent with other similar scenarios. Also, the usage is confined to only two places.